.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Kubernetes-sigs/krew CI
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
+
+jobs:
+
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    
+    - name: Set GOPATH
```

Comment:
the name isn't reflecting what's happening below.
It sounds more like "Add GOPATH/bin to PATH"

Reply:
Sure, It's more clear.